src/ComponentConfig.js

```diff
@@ -54,17 +54,79 @@ module.exports = class ComponentConfig {
 
         return Object.entries(services).reduce((newConfig, [serviceName, config]) => {
             const normalizedConfig = this.normalizeService(serviceName, config);
+            // eslint-disable-next-line no-param-reassign
             newConfig[serviceName] = normalizedConfig;
             return newConfig;
         }, {});
     }
 
+    static modelIsExposed(name, modelsConfig) {
+        // if there is no configuration all models are exposed
+        if (!modelsConfig) {
+            return true;
+        }
+        const config = modelsConfig[name];
+        // false or not set
+        if (!config) {
+            return false;
+        }
+        if (config === true) {
+            return true;
+        }
+        return config.expose !== false;
```

Comment:
Are you intentionally setting the value to true if nothing is configured?
eg.: Would lead to "expose": true
```
{
     "isGlobal": false
     "isPublic": false,

}
```

Reply:
Expose within the service means that it is available and true is the default. The resulting config

```Javascript
{
     expose: true,
     isGlobal: false,
     isPublic: false,
}
```

perfectly makes sense: If you only want to consume the model _e.g._ in a controller. This is the best way to configure it!